src/ec/packages/ec-component-datepicker/ecl-datepicker.html.twig

```diff
@@ -34,6 +35,7 @@
 {% set _placeholder = placeholder|default('dd/mm/yyyy') %}
 {% set _icons_path = icons_path|default('') %}
 {% set _invalid = invalid|default(false) %}
+{% set _invalid_text = invalid_text|default('') %}
```

Comment:
This is missing in the documentation, instead (invalid_text), but only in the template, the README has it, helper_text is also missing in the template docs